### ISSUE TITLE
[JS]: Missing documentation on running the firstscript.spec.js

### ISSUE THREAD

**Issue by N1lshui** [SeleniumHQ/seleniumhq.github.io#1160]
### What happened?

I'm not able to run Slenium with Javascript. I try to start my tes.js an dthere is this error

### How can we reproduce the issue?

```shell
I don't know
test.js
```

### Relevant log output

```shell
$ node test.js
[INFO] Searching for WebDriver executables installed on the current system...
[INFO] ... located firefox
[INFO] Running tests against [firefox]
C:\Users\nbirken\Documents\Selenium\js\node_modules\selenium-webdriver\testing\index.js:229
  after(function () {
  ^

ReferenceError: after is not defined
    at init (C:\Users\nbirken\Documents\Selenium\js\node_modules\selenium-webdriver\testing\index.js:229:3)
    at suite (C:\Users\nbirken\Documents\Selenium\js\node_modules\selenium-webdriver\testing\index.js:374:5)
    at Object.<anonymous> (C:\Users\nbirken\Documents\Selenium\js\test.js:5:1)
    at Module._compile (node:internal/modules/cjs/loader:1105:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1159:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Function.executeUserEntryPoint [as runMain] (node:internal/modules/run_main:77:12)
    at node:internal/main/run_main_module:17:47
```

### Operating System

Windows 10

### Selenium version

4.0.0

### What are the browser(s) and version(s) where you see this issue?

Firefox

### What are the browser driver(s) and version(s) where you see this issue?

Firefox

**Comment by harsha509** [comment]
Hi @N1lshui ,


Thank you for raising the issue. 

Yes there is a missing info in document on running the test. 
To execute, please install mocha by `npm install mocha` and run test using `mocha test.js`

As this is not an issue with selenium-webdriver i will move this issue to documentation!!!

Thanks,
Sri Harsha

**Comment by dublinsubway** [comment]
I can tackle this.

Would it be reasonable to add a last step to run the script [here](https://www.selenium.dev/documentation/webdriver/getting_started/first_script/)?

**Comment by titusfortner** [comment]
You can add a comment at the bottom there in the JS tab for how to execute the code: https://github.com/SeleniumHQ/seleniumhq.github.io/blob/trunk/website_and_docs/content/documentation/webdriver/getting_started/first_script.en.md?plain=1#L243
(needs to be updated in all 4 language files)

Can also point people to our example repo for JS.... https://github.com/SeleniumHQ/seleniumhq.github.io/tree/trunk/examples/javascript
Which... prompts #1174 since it should be easy for a user to see how to run those on their machine.

Thanks for your help!

**Comment by dublinsubway** [comment]
> Can also point people to our example repo for JS.... https://github.com/SeleniumHQ/seleniumhq.github.io/tree/trunk/examples/javascript
> Which... prompts #1174 since it should be easy for a user to see how to run those on their machine.

There already is a link to see the code on GitHub at the bottom of the tab.
![image](https://user-images.githubusercontent.com/61555192/193363367-32834718-c433-415b-a6f1-3c28244b4c71.png)
Do you think it's not enough?

I put the text in other languages in plain English as I, unfortunately, lack language skills, but since quite a chunk of Japanese page is in English anyway, it's acceptable, hopefully.

**Comment by reply2sagar** [comment]
Can someone assign this issue to me please? I will work on this. Or do I need to create PR directly?

**Comment by harsha509** [comment]
Hi @N1lshui,

The steps to run JS code have been updated in our documentation.

Closing this as resolved now.
